lib/xslt/fgdc2geoBL.xsl

```diff
@@ -52,42 +63,40 @@
         </xsl:for-each>
     </xsl:variable>
   
-  <!-- Output bounding box -->
+<!-- Output bounding box -->
 <xsl:variable name="upperCorner">
     <xsl:value-of select="number(metadata/idinfo/spdom/bounding/eastbc)"/>
-  <xsl:text> </xsl:text>
+    <xsl:text> </xsl:text>
     <xsl:value-of select="number(metadata/idinfo/spdom/bounding/northbc)"/>
 </xsl:variable>
   
-  <xsl:variable name="lowerCorner">
-
-      <xsl:value-of select="number(metadata/idinfo/spdom/bounding/westbc)"/>
-      <xsl:text> </xsl:text>
-      <xsl:value-of select="number(metadata/idinfo/spdom/bounding/southbc)"/>
-
-  </xsl:variable>
-  <xsl:variable name="x2" select="number(metadata/idinfo/spdom/bounding/eastbc)"/><!-- E -->
-  <xsl:variable name="x1" select="number(metadata/idinfo/spdom/bounding/westbc)"/><!-- W -->
-  <xsl:variable name="y2" select="number(metadata/idinfo/spdom/bounding/northbc)"/><!-- N -->
-  <xsl:variable name="y1" select="number(metadata/idinfo/spdom/bounding/southbc)"/><!-- S -->
-
+ <xsl:variable name="lowerCorner">
+     <xsl:value-of select="number(metadata/idinfo/spdom/bounding/westbc)"/>
+     <xsl:text> </xsl:text>
+     <xsl:value-of select="number(metadata/idinfo/spdom/bounding/southbc)"/>
+</xsl:variable>
   
+  <xsl:variable name="x2" select="substring-before($upperCorner/text(), ' ')"/><!-- E -->
```

Comment:
Ln 79 - 82 were pulled out for a specific reason. Using this xpath doesn't work within Nokogiri, so I modified the way the original fgdc2geobl worked.